run_and_analyze.sh

```diff
@@ -96,6 +96,21 @@ case "$scheduler" in
         collective_string=$(echo $collectives | sed 's/ /,/g')
         qsub -hold_jid $collective_string $work_dir/analyze.sh -c $config_file
     ;;
+    "PBS")
```

Comment:
This doesn't automatically analyze the results like for the other schedulers. Not a big deal though since the analyze script doesn't take long to run...Might separate the functionality later.

Reply:
Yes, I left the automatic analysis open for the moment. With PBS it is not so easily to reproduce the same behaviour with the job dependencies, here. So I would prefer to separate this as well to keep this part of the code simple.